test/cluster/config_ca_test.go

```diff
@@ -0,0 +1,147 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cluster
+
+import (
+	"crypto/tls"
+	"encoding/pem"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/hyperledger-labs/orion-server/pkg/server/testutils"
+	"github.com/hyperledger-labs/orion-server/pkg/types"
+	"github.com/hyperledger-labs/orion-server/test/setup"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+// Scenario:
+// - try to update CertAuthConfig to empty CAConfig => the tx fails
+// - add Root CA & Intermediate CA to CertAuthConfig
+func TestUpdateCA(t *testing.T) {
```

Comment:
Add some negative test cases, i.e. 
- Adding an intermediate CA without adding the root it came from in the same Tx (broken chain)
- Try to add a bad CA cert
Also: 
- Remove a CA (after having added a new one) etc.

Reply:
done